### ISSUE TITLE
Add some indexes as a benchmark

### ISSUE THREAD

**Issue by casitu** [ghostfolio/ghostfolio#806]
I think it would be nice, maybe in the summary page of the overview, to add a couple of benchmarks (or even customize which ones through your preferences) such as S&P500 or EuroStoxx50 to confront the annualized performance of your portfolio vs the annualized performance of the benchmark.

**Comment by dtslvr** [comment]
Yes, that would be a great addition. Are you interested in implementing the benchmark feature @casitu?

**Comment by casitu** [comment]
I would love to help, yeah. Coding is not my strongest suit, but let me collect some ideas and I'll post them here. Let me know how can I be of assistance.

**Comment by casitu** [comment]
I believe main international indexes are:

- Asia: SSE Composite Index, Hang Seng and Nikkei 225
- Europe: CAC 40, DAX 30 and FTSE 100
- America: Dow Jones, S&P 500, Nasdaq 100 and Bovespa
- Global: MSCI World

I'd create a new page under "Portfolio" section called "Benchmarking" to compare the users' annualized performance (as calculated in the summary) against those indexes. A table displaying user's performance and indexes' performances.

Because you have the date of the first activity of the user, a call to a webservice to get the index's value in that particular day and current value and a simple calculation to get the annualized performance.
